### PR TITLE
[4.1] Correct return value type

### PR DESCRIPTION
**Pull request by heelc29**
### Summary of Changes

Correct return value type for function getListQuery in ListModel
`string` → `\Joomla\Database\DatabaseQuery`

Adjust spaces in docblock

### Testing Instructions

Code review

### Documentation Changes Required

None

### CODE REVIEW

File: administrator/components/com_tags/src/Model/TagsModel.php [hunk 1]
```diff
@@ -29,11 +29,10 @@ class TagsModel extends ListModel
 	/**
 	 * Constructor.
 	 *
-	 * @param   MVCFactoryInterface  $factory  The factory.
+	 * @param  MVCFactoryInterface  $factory  The factory.
+	 * @param  array                $config   An optional associative array of configuration settings.
```
Comment (Quy) [inline administrator/components/com_tags/src/Model/TagsModel.php]:
```suggestion
	 * @param  array                $config   An optional associative array of configuration settings.
	 * @param  MVCFactoryInterface  $factory  The factory.
```

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on e13be75584e567ebf3b716b36f10b98ac6e24195<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/36805">issues.joomla.org/tracker/joomla-cms/36805</a>.</sub>

**Comment by HLeithner** [comment]
This pull request has automatically rebased to 4.2-dev.

**Comment by laoneo** [comment]
Can you fix the conflicts here?

**Comment by heelc29** [comment]
All done in #39990

**Pull request closed without merging**